src/Events/Startup.cs

```diff
@@ -70,6 +70,9 @@ public void ConfigureServices(IServiceCollection services)
             services.AddSingleton<IEventWriteService, RepositoryEventWriteService>();
         }
 
+        // To be removed in AC-1723 provided feature flags are not otherwise being used in this project
```

Comment:
❓ Is this necessary? Doesn't hurt to load the service in my opinion, for future use.

Reply:
I wasn't sure how aggressive to be about that, but I agree. Comment removed.